administration/src/bp-modules/regions/application-confirmation-note/ApplicationConfirmationNote.tsx

```diff
@@ -0,0 +1,61 @@
+import { Checkbox } from '@blueprintjs/core'
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import TextAreaDialog from '../../components/TextAreaDialog'
+
+const StyledCheckbox = styled(Checkbox)`
+  margin: 12px;
+`
+
+type ApplicationConfirmationNoteDialogProps = {
+  defaultConfirmationNote?: string | null
+  defaultConfirmationNoteActivated: boolean
+  isOpen: boolean
+  isSavingApplicationConfirmationNote: boolean
+  setIsOpen: (value: boolean) => void
+  saveApplicationConfirmationNote: (text: string, activated: boolean) => void
+}
+
+const MAX_NOTE_CHARS = 1000
```

Comment:
🙃 as I understood it should be something really short, like 1-2 sentences, so 500 might be enough? 
(not so important actually)

Reply:
hm there was no clear requirement. i think it does not harm to have 1000